src/swarm/neo/client/mixins/RequestCore.d

```diff
@@ -190,13 +195,28 @@ public template RequestCore ( RequestType request_type_, ubyte request_code,
 
     public static struct Context
     {
+        import ocean.util.serialize.contiguous.Serializer;
+        import ocean.util.serialize.contiguous.Deserializer;
+
+        private void[] user_specified_params_blob;
```

Comment:
` However, Contiguous.data is not a public field so the de/serialiser would mess with private property`

`copy` does it for you - https://github.com/sociomantic-tsunami/ocean/blob/v2.x.x/src/ocean/util/serialize/contiguous/Util.d#L68 (though `const` fix for `copy` was merged very recently and may not be available for swarm :X)